packages/web-components/src/custom-elements.ts

```diff
@@ -129,18 +129,18 @@ export const allComponents = {
   fluentTreeView,
   fluentTreeItem,
   register(container?: Container) {
-      if (!container) {
```

Comment:
is this intentional? looks like a change outside of react-northstar

Reply:
Thanks for catching this @jurokapsiar  This code has nothing to do with React. The web components packages should not be changed without approval from @chrisdholt

Reply:
Oops, not sure how these changes got in here. haven't ever (intentionally) touched these files.